packages/neo-one-build-tests/jest/e2e.js

```diff
@@ -6,11 +6,7 @@ module.exports = {
   testRunner:
     '<rootDir>/packages/neo-one-build-tests/node_modules/jest-circus/runner',
   testRegex: '^.*/__e2e__/.*\\.test\\.tsx?$',
-  moduleNameMapper: {
-    '^@neo-one/ec-key': '@neo-one/ec-key',
-    '^@neo-one/boa': '@neo-one/boa',
-    '^@neo-one/csharp': '@neo-one/csharp',
-  },
+  moduleNameMapper: undefined,
```

Comment:
I don't think I've ever touched `e2e` tests, so curious again, but why remove the `moduleNameMapper` here?